### PR TITLE
BSDT-25: Сheck the title, url, image on the brand page

### PR DESCRIPTION
**Pull request by Dimas0006**

### CODE REVIEW

File: src/test/java/tests/СheckingTitleBrandTest.java [hunk 1]
```diff
@@ -5,26 +5,27 @@
...
 import pages.FindYourStoreTeamPage;
 import pages.HomePage;
-import pages.StoreBrand;
+import pages.StoreBrandPage;
 
 public class СheckingTitleBrandTest extends AbstractBaseTest {
 
     @Test
-    public void CheckTitleBrand() {
+    public void checkTitleBrand() {
```
Comment (ArtemKostiuktest) [inline src/test/java/tests/СheckingTitleBrandTest.java]: please delete empty line

File: src/test/java/tests/СheckingTitleBrandTest.java [hunk 2]
```diff
@@ -5,26 +5,27 @@
...
+
+        findYourStoreTeamPage
+                .setInputWhatYourTeam("FLX SEALS ATHLETICS")
+                .clickTeamCard();
 
         Assert.assertEquals(homePage.getUrl(), "https://teamlocker.squadlocker.com/#/lockers/flx-seals-athletics");
-        Assert.assertEquals(storeBrand.getTitleBrand().getText().trim(), "FLX SEALS ATHLETICS");
-        Assert.assertTrue(storeBrand.getImgBrand().isDisplayed());
+        Assert.assertEquals(storeBrandPage.getTitleBrand().getText().trim(), "FLX SEALS ATHLETICS");
+        Assert.assertTrue(storeBrandPage.getImgBrand().isDisplayed());
```
Comment (ArtemKostiuktest) [inline src/test/java/tests/СheckingTitleBrandTest.java]: pleace delete empty line

**Review by ArtemKostiuktest — APPROVED** [review]

File: src/test/java/pages/FindYourStoreTeamPage.java [hunk 3]
```diff
@@ -12,15 +10,16 @@ public FindYourStoreTeamPage(WebDriver driver) {
...
     }
 
-    private static final String TEAM_CARD = "//a[@class='team-cart']";
-
-    public WebElement getTeamCard() {
-        return waitUntilElementToBeClickable(TEAM_CARD);
+    public FindYourStoreTeamPage clickTeamCard() {
+        waitUntilElementToBeClickable(TEAM_CARD).click();
+        return this;
     }
```
Comment (CheriakYuriy123) [inline src/test/java/pages/FindYourStoreTeamPage.java:24]: please delete empty line

File: src/test/java/tests/СheckingTitleBrandTest.java [hunk 4]
```diff
@@ -5,26 +5,27 @@
...
 import pages.FindYourStoreTeamPage;
 import pages.HomePage;
-import pages.StoreBrand;
+import pages.StoreBrandPage;
 
 public class СheckingTitleBrandTest extends AbstractBaseTest {
 
-    @Test
-    public void CheckTitleBrand() {
+    private static final String URL_PAGE_BRAND = "https://teamlocker.squadlocker.com/#/lockers/flx-seals-athletics";
```
Comment (AndriiMyronenko91) [inline src/test/java/tests/СheckingTitleBrandTest.java:12]: please rename to BRAND_PAGE_URL

File: src/test/java/tests/СheckingTitleBrandTest.java [hunk 5]
```diff
@@ -5,26 +5,27 @@
...
+    private static final String TITLE_BRAND = "FLX SEALS ATHLETICS";
```
Comment (AndriiMyronenko91) [inline src/test/java/tests/СheckingTitleBrandTest.java:13]: BRAND_TITLE

**Review by AndriiMyronenko91 — CHANGES_REQUESTED** [review]

**Review by AndriiMyronenko91 — APPROVED** [review]

**Pull request merged**